composite-api/README.md

```diff
@@ -0,0 +1,75 @@
+# Composite API
+
+---
+This sample shows how to implement composite APIs (e.g. mashup) using Apigee's [ServiceCallout Policy](https://cloud.google.com/apigee/docs/api-platform/reference/policies/service-callout-policy) and [ExtractVariables Policy](https://cloud.google.com/apigee/docs/api-platform/reference/policies/extract-variables-policy)
+
+Let's get started!
+
+---
+
+## Prerequisites
+
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro)
+2. Configure [external access](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing#external-access) for API traffic to your Apigee X instance
+3. Access to deploy proxies, create products, apps and developers in Apigee
```

Comment:
This sample does not need Products, Developers and Apps. So lets just say "Access to deploy proxies"